app/models/assignment_form.rb

```diff
@@ -2,7 +2,13 @@
 require 'active_support/time_with_zone'
 
 class AssignmentForm
-  attr_accessor :assignment, :assignment_questionnaires, :due_dates, :tag_prompt_deployments, :is_conference_assignment
+  attr_accessor :assignment,
+                :assignment_questionnaires,
+                :due_dates,
+                :tag_prompt_deployments,
+                :is_conference_assignment,
+                :auto_assign_mentor
+
   attr_accessor :errors
```

Comment:
any reason why this is not with the comma separated list above?

Reply:
Not that I know of. It was that way already, but we can move it up there.

Reply:
yeah combining it would be good.

Reply:
Done!

Reply:
This change seems to have caused the Travis build to fail. Not sure how that would come to be.